src/rust/common/src/signal.rs

```diff
@@ -0,0 +1,8 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+#[derive(Clone)]
+pub enum Signal {
```

Comment:
What's the story with logrotate, currently for debug log only I guess?

Reply:
Still need to work on the logging story - currently we don't even have file logging 🙃